packages.el

```diff
@@ -60,12 +60,19 @@
 (package! ob-sql-mode)
 (package! ob-tmux)
 (package! ox-gfm) ; org dispatch github flavoured markdown
+;; :build (:not compile) so we can edit in place
```

Comment:
I found it a pain to need to doom sync / doom compile on these very small files.
So I relaxed the byte compilation requirement.